### DISCUSSION TITLE
Analytics for sites without frameworks

### DISCUSSION THREAD

**Discussion by devxan** [vercel/community#306] [Help] [answered]
My [site](https://devxan.gq/) is a simple HTML site without a framework. Could I use analytics in the future?

**Comment by leerob** [comment, marked as answer]
Hey! This is now possible using the [Vercel Analytics API](https://vercel.com/docs/concepts/analytics/api). You can view similar examples for [SvelteKit](https://github.com/vercel/vercel/tree/main/examples/sveltekit) and [Create React App](https://github.com/vercel/vercel/tree/main/examples/create-react-app) to see how it's implemented.
